atlasdb-client/src/test/java/com/palantir/atlasdb/keyvalue/impl/TracingKeyValueServiceTest.java

```diff
@@ -120,7 +117,7 @@ public void addGarbageCollectionSentinelValues() throws Exception {
         ImmutableSet<Cell> cells = ImmutableSet.of(CELL);
         kvs.addGarbageCollectionSentinelValues(TABLE_REF, cells);
 
-        checkSpan("atlasdb-kvs.addGarbageCollectionSentinelValues(test.testTable, 1 cells)");
+        checkSpan("atlasdb-kvs.addGarbageCollectionSentinelValues({table}, 1 cells)");
```

Comment:
can we have one sanity check that it actually includes the table name if it's safe?

Reply:
I addressed all the comments but this one. Turns out that testing if a table is safe or not is not that easy, as that requires some mocking/setup, and the classes to mock are package-private. To address this I added testing on the API we use on LoggingArgsTest.